cdc/sink/dmlsink/mq/kafka_dml_sink.go

```diff
@@ -122,7 +122,7 @@ func NewKafkaDMLSink(
 	metricsCollector := factory.MetricsCollector(tiflowutil.RoleProcessor, adminClient)
 	dmlProducer := producerCreator(ctx, changefeedID, asyncProducer, metricsCollector, errCh, failpointCh)
 	concurrency := tiflowutil.GetOrZero(replicaConfig.Sink.EncoderConcurrency)
```

Comment:
since `SinkConfig` is passed to the encoder group, `concurrency` can be moved inside `NewEncoderGroup`